samples/kubernetes/helm/fhir-server/templates/deployment.yaml

```diff
@@ -0,0 +1,127 @@
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: ASPNETCORE_FORWARDEDHEADERS_ENABLED
+              value: "true"
+            {{- if .Values.appInsights.secretName }}
+            - name: ApplicationInsights__InstrumentationKey
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.appInsights.secretName }}
+                  key: {{ .Values.appInsights.secretKey }}
+            {{- end }}
+            - name: FhirServer__Security__Enabled
+              value: {{ .Values.security.enabled | quote }}
+            - name: FhirServer__Security__EnableAadSmartOnFhirProxy
+              value: {{ .Values.security.enableAadSmartOnFhirProxy | quote }}
+              {{- if .Values.security.authority }}
+            - name: FhirServer__Security__Authentication__Authority
+              value: {{ .Values.security.authority | lower | quote }}
+              {{- end }}
+              {{- if .Values.security.audience }}
+            - name: FhirServer__Security__Authentication__Audience
+              value: {{ .Values.security.audience | lower | quote }}
+              {{- end }}
+            - name: DataStore
+              value: {{ .Values.database.dataStore }}
+            {{- if eq .Values.database.dataStore "SqlServer" }}
+            - name: SqlServer__AllowDatabaseCreation
+              value: "true"
+            - name: SqlServer__Initialize
+              value: "true"
+            - name: DATABASESERVERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "fhir-server.fullname" . }}
+                  key: fullyQualifiedServerName
+            - name: DATABASEUSERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "fhir-server.fullname" . }}
+                  key: username
+            - name: DATABASEPASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "fhir-server.fullname" . }}
+                  key: password
+            - name: DATABASECATALOG
+              value: {{ include "fhir-server.fullname" . }}
+            - name: MAX_POOL_SIZE
+              value: {{ .Values.database.sql.maxPoolSize | quote }}
+            - name: SqlServer__ConnectionString
+              value: "Server=tcp:$(DATABASESERVERNAME),1433;Initial Catalog=$(DATABASECATALOG);Persist Security Info=False;User ID=$(DATABASEUSERNAME);Password=$(DATABASEPASSWORD);MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;Max Pool Size=$(MAX_POOL_SIZE)"
+            {{- end }}
+            {{- if eq .Values.database.dataStore "CosmosDb" }}
+            - name: COSMOSDBSERVER
```

Comment:
what is this used for?

Reply:
Good catch, that is a left-over from a previous version of the azure service operator api.